src/components/NotificationsDrawer/DrawerPanel.tsx

```diff
@@ -0,0 +1,515 @@
+import React, { useEffect, useMemo, useState } from 'react';
+import { PopoverPosition } from '@patternfly/react-core/dist/dynamic/components/Popover';
+import { Badge } from '@patternfly/react-core/dist/dynamic/components/Badge';
+import { useAtom, useAtomValue, useSetAtom } from 'jotai';
```

Comment:
We are using redux in notifications application, so it might be better to change the state logic to it so we don't have 2 different state management systems.